### ISSUE TITLE
variables3.rs satisfied with no mut variables.

### ISSUE THREAD

**Issue by EvanCarroll** [rust-lang/rustlings#397]
It's clear from the hint but not from the instruction that we're supposed to use `mut` here,

The code is 

```rust
fn main() {
    let x = 3;
    println!("Number {}", x);
    x = 5;
    println!("Number {}", x);
}
```

But, this can be solved with shadowing,

```rust
fn main() {
    let x = 3;
    println!("Number {}", x);
    let x = 5;
    println!("Number {}", x);
}
```

Perhaps we can find a better way to check for this. Or to instruct the user to make this compile without changing the line 

    let x = 5;

Like,

```rust
fn main() {
    let x = 3;
    println!("Number {}", x);
    let x = 5;   // don't change this line
    println!("Number {}", x);
}
```

We can even force both methods here,

```rust
fn main() {
    let x = 3;
    println!("Number {}", x);
    x = 5;  // don't change this line
    println!("Number {}", x);

    let y = 3;  // don't change this line
    println!("Number {}", y);
    y = 5;
    println!("Number {}", y);
}
```

**Comment by jrvidal** [comment]
Yeah, shadowing is a tricky concept to introduce vs. mutability. I lean towards a "Don't change this line" to focus on mutability.

**Comment by jrvidal** [comment]
Closed in #408
